warp/ChangeLog.md

```diff
@@ -1,3 +1,9 @@
+## 3.2.28
```

Comment:
Looks like this can be included in 3.2.27, since that version hasn't been released to Hackage yet.

Reply:
I saw a discrepancy between changelog and cabal, so I raised it a bit more. 
Should I revert it or will you?